packages/jetpack-redirects/README.md

```diff
@@ -0,0 +1,56 @@
+# Jetpack Redirects
+
+A helper function to build URLs using the `jetpack.com/redirect` service.
+
+If you are an automattician, refer to PCYsg-pY7-p2 for all related information and links to our internal tools.
+
+## Usage
+
+`getRedirectUrl( required source, optional args );`
+
+### source (required)
+
+Source can be either a “source handler” or a URL.
+
+A “source handler” must be registered in the Jetpack Redirects service, on the server side. It’s a slug that points to an URL that may or may not have dynamic parts in it.
+
+A “URL” is a string that must start with “https://" and doesn’t need to be registered on the server. However, if it is registered, it will point to the URL set as target there rather than to the source. (Note: It will only work for whitelisted domains)
+
+### args (optional)
+
+This is optional and allows you to pass an array (or object in JS) with more parameters to build the URL:
```

Comment:
In JS, only object is allowed, is that right? Then let's not confuse JS developers by PHP references, and doing that even without mentioning the PHP language 🙂

Reply:
Tried to make it all more clear in https://github.com/Automattic/wp-calypso/pull/41497/commits/c733a134a39f478a2aa993b814acf9d464ca8719